### ISSUE TITLE
Filter option does not appear for strings and keywords fields

### ISSUE THREAD

**Issue by rjurado01** [appbaseio/dejavu#50]
I use elasticsearch 5.0.0 with this index:

```
curl -XPUT http://localhost:9200/cnaes -d '
{
 "mappings" : {
  "_default_" : {
   "properties" : {
    "code" :          {"type": "keyword" },
    "descripcion" :   {"type": "string" },
    "value":          {"type": "integer"}
   }
  }
 }
}
';
```

When I access to Dejavu and inspect index, filter option does not appear for strings and keywords fields.

![dejavu_screenshot](https://cloud.githubusercontent.com/assets/1703608/22062841/1fa777ca-dd7c-11e6-910a-9195ec343bcd.png)

Any help?

**Comment by siddharthlatest** [comment]
@rjurado01 We should be adding support for this in the next release.

**Comment by rjurado01** [comment]
@siddharthlatest :thumbsup:

**Comment by farhan687** [comment]
Fixed here: e044e25e547e5d76ce224a76b20f8ce6cc2b38ba

**Comment by rjurado01** [comment]
@farhan687 Thanks!!
